data/readreplica.py

```diff
@@ -97,6 +117,10 @@ def _select_database(cls):
         if cls._meta.database.transaction_depth() > 0:
             return cls._meta.database
 
+        # Select if forced.
+        if getattr(cls._meta.database._state, _FORCE_MASTER_COUNTER_ATTRIBUTE, 0):
```

Comment:
I don't like implicit falsiness in Python. Please add the `> 0` to the end of this.

Reply:
Done